### ISSUE TITLE
Support for deploy: labels

### ISSUE THREAD

**Issue by flaviostutz** [docker/compose-cli#776]
ECS has Tags semantics for containers. We could set Service Tags with the contents of docker-compose labels.

I have a use case where I currently use labels for configuring some external tasks in Swarm, and I would like to move this behavior to ECS.

**Comment by ndeloof** [comment]
ECS both support *container labels* by task definition (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-containerdefinitions.html#DockerLabels) and *tags* on most (but not all) components, which we set when possible by `com.docker.compose.project` to make it easier filtering stack resources - we could create a ResourceGroup to make this easier to use.

Can you please clarify your use-case?

**Comment by flaviostutz** [comment]
I have a bunch of labels on my docker-compose.yml but when deploying it to ECS I don't know where they are placed. Do docker-compose currently copies the docker-compose.yml labels to ECS DockerLabels? Maybe I'm missing something...

**Comment by ndeloof** [comment]
So my question regarding your use case. On ECS a 'service' is actually backed by a bunch of aws components, some of them could be tagged with  composefile labels, and/or we can just set ECS task's Docker labels, but for the latter, without any API that you can rely on to use those, I hardly can imagine the benefits.

**Comment by flaviostutz** [comment]
I was thinking about creating a Caddy service that would listen to those service (or even task) labels and configure itself, just like works with Docker Swarm here (https://github.com/lucaslorentz/caddy-docker-proxy) - we use this in production and works like a charm.

https://github.com/lucaslorentz/caddy-docker-proxy/issues/195

**Comment by ndeloof** [comment]
@flaviostutz thanks for the details. As I guesed for such a scenario DockerLabels would be of no help, you'd need ECS service tags set, and use AWS specific API to search for them. But you also would need your Security Groups setup so they allow traffic with Caddy server.

Such a (non-trivial) setup is exactly what we implemented using AWS Load Balancer (Listeners, TargetGroups) so this would come as a duplicate infrastructure for the same needs, wouldn't it?
